dll/win32/shell32/CDefView.cpp

```diff
@@ -574,6 +576,12 @@ BOOL CDefView::CreateList()
     if (!m_ListView)
         return FALSE;
 
+    if (!(m_FolderSettings.fFlags & FWF_DESKTOP))
+    {
+        const DWORD dwLBExStyle = LVS_EX_TRANSPARENTBKGND;
+        ::SendMessage(m_ListView, LVM_SETEXTENDEDLISTVIEWSTYLE, dwLBExStyle, dwLBExStyle);
```

Comment:
Might be better to first retrieve the current state, and then or the flag onto that?

Reply:
This message process uses mask. No retrieving required.

Reply:
Ok, I did not know this.